CHANGELOG.md

```diff
@@ -1,8 +1,10 @@
 # Change Log
 
-* New: Add EXEMultiTunerPlugin, HCWIRBlaster & USB-UUIRT VS projects (Windows)
-* 64-bit code and VS project updates (Windows)
-* Enabled MSYS2/MinGW compile for FFMPEG-based projects (Windows)
+* Change: New installation properties default video/dvd_video renderer is 'EVR'
+* Change: New installation properties default video/audio decoder is 'Default'

```

Comment:
```suggestion
* Change: New installation properties default video/audio decoder are auto-detected
```

Reply:
I'll add that and the Win10 VISTA_OS detection to the change log.